ideas/xxx-sob-improve-onboarding.md

```diff
@@ -0,0 +1,77 @@
+## Preamble
+
+    Idea: #XX-sob-improve-onboarding
+    Title: OpenBounty Improved Onboarding
+    Status: In progress
+    Created: 2018-04-04
+
+## Summary
+
+Currently users wanting to publish a bounty need to go through various manual steps, including us whitelisting them for using the platform. This step forces users to wait for us and might make them drop out of the process even if they were committed to create a bounty.
+
+This swarm aims to remove the need for whitelisting users and revamp the onboarding flow at the same time.
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch (10hr for MVP, 30hr+ from MVP onwards)
+- Testing & Evaluation: @Churik
+- UX Contributor: @EugeOrtiz 
+- Dev Contributor: @siphuel or @rcullito
+- Contracts Contributor: @3esmit
+
```

Comment:
As a general comment is the purpose of this swarm the removal of the whitelisting process for bounty creators, or an overall better process for anyone using the platform?

Are key results 3/4 being tackled in a different swarm?

Reply:
> As a general comment is the purpose of this swarm the removal of the whitelisting process for bounty creators, or an overall better process for anyone using the platform?

Removal of whitelisting requirement and revamped onboarding for anyone using the platform.

> Are key results 3/4 being tackled in a different swarm?

> - KR3: Create FAQs and “How-to” videos & integrate into site/app
> - KR4: Ship funnel metrics for bounty issuers and bounty hunters

Should be tackled in different swarms IMHO. Maybe KR4 could be folded into the metrics one too.